tests/test_service.py

```diff
@@ -74,3 +75,26 @@ def test_connect_one_to_one_to_one(self):
                 'receivers': ['three']
             }
         )
+
+    def test_command(self):
+        cmd_structure = '{}/{}/{}'.format
+        s = Service('name', 'type')
+        blk = TestBlock('one')
+        s.connect(blk, TestBlock('two'))
+        s._instance = lambda: None
+        mm = MagicMock()
+        s._instance._get = mm
+
+        s.command('foo')
+        assert mm.called
```

Comment:
I believe this sort of assert statement is meant for use in code. Unit tests have specific methods to be used. See existing tests for examples.